src/app/pages/SearchResultsPage/ResultsOnNetwork.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react'
+import { getNetworkNames, Network } from '../../../types/network'
+import { useTranslation } from 'react-i18next'
+import { useTheme } from '@mui/material/styles'
+import { getThemesForNetworks } from '../../../styles/theme'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { NetworkThemeBubble } from '../../components/NetworkThemeBubble'
+import Box from '@mui/material/Box'
+import { SearchResultsList } from './SearchResultsList'
+import { SearchQueries } from './hooks'
+import Typography from '@mui/material/Typography'
+
+export const ResultsOnNetwork: FC<{
+  network: Network
```

Comment:
Ah, this is network from results, not from search query. Rename? Comment?

Reply:
OK, adding comments to all similar components.